pop/auth.c

```diff
@@ -343,10 +348,11 @@ static enum PopAuthRes pop_auth_oauth(struct PopAccountData *adata, const char *
   snprintf(auth_cmd, auth_cmd_len, "AUTH OAUTHBEARER %s\r\n", oauthbearer);
   FREE(&oauthbearer);
 
+  const short c_debug_level = cs_subset_number(NeoMutt->sub, "debug_level");
   int ret = pop_query_d(adata, auth_cmd, strlen(auth_cmd),
 #ifdef DEBUG
                         /* don't print the bearer token unless we're at the ungodly debugging level */
-                        (C_DebugLevel < MUTT_SOCK_LOG_FULL) ?
+                        (c_debug_level < MUTT_SOCK_LOG_FULL) ?
```

Comment:
```suggestion
                        (cs_subset_number(NeoMutt->sub, "debug_level") < MUTT_SOCK_LOG_FULL) ?
```

To pet the "unused variable" warning in non-debug builds.